wooting-macro-backend/src/plugin/delay.rs

```diff
@@ -1,2 +1,13 @@
+#[cfg(target_os = "windows")]
+pub const STANDARD_KEYPRESS_DELAY: u64 = 1;
+
+#[cfg(target_os = "macos")]
+pub const STANDARD_KEYPRESS_DELAY: u64 = 10;
+
+#[cfg(target_os = "linux")]
+pub const STANDARD_KEYPRESS_DELAY: u64 = 10;
```

Comment:
Aren't these numbers relevant in the frontend as well? But I don't remember seeing any platform specific handling for this in the defaults

Reply:
This is slightly different. The numbers here correspond to the actual queue being pushed to the OS and that has it's own delays. 

This is basically the delay if multiple macros are pressed so they don't collide with each other, whereas the delay in the frontend is per-action macro delay. 

Not sure how I could unify these though